src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -367,14 +366,15 @@ export class WebRTCAdaptor
 		}
 	};
 
-	enableMicInMixedAudio(enable) 
+	enableSecondStreamInMixedAudio(enable) 
 	{
-		if (this.micGainNode != null) {
+		
+		if (this.secondStreamGainNode != null) {
 			if (enable) {
-				this.micGainNode.gain.value = 1;
+				this.secondStreamGainNode.gain.value = 1;
 			}
 			else {
-				this.micGainNode.gain.value = 0;
+				this.secondStreamGainNode.gain.value = 0;
```

Comment:
This seems like it is being overridden by mixAudioStreams everytime since inside that it the secondStreamGainNode is hardcoded into 1.

Reply:
You're right. Let me fix it.

Reply:
I've take a look at again and it seems that it's meaningful to set the gain node to 1 when mixing the stream.  
User can disable/enable it with `enableSecondStreamInMixedAudio` method.